sale_order_import/tests/test_sale_order_import.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import base64
+import os
+
+from odoo.tests import common
+from odoo.tools.misc import file_open
+from odoo.tools.safe_eval import safe_eval
+
+
+class TestSaleOrderImport(common.TransactionCase):
+    """
+    This tests allows to import the test
+    sale order csv file and import sale order.
```

Comment:
@YogeshMahera-SerpentCS Not clear what you are trying to say with this sentence.